src/System.Runtime.Extensions/tests/Performance/Perf.Random.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Xunit.Performance;
+
+namespace System.Runtime.Extensions.Tests
+{
+    public class Perf_Random
+    {
+        [Benchmark]
+        public void ctor()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    new Random(); new Random(); new Random();
+                    new Random(); new Random(); new Random();
+                    new Random(); new Random(); new Random();
+                }
+        }
+
+        [Benchmark]
+        public void Next_int()
+        {
+            Random rand = new Random();
```

Comment:
For a bit more consistency on these Next tests, I'd suggest constructing the Random with a fixed seed.